sw/device/silicon_creator/lib/crypto/rsa_3072/rsa_3072_verify.s

```diff
@@ -52,3 +52,33 @@ verify:
 .balign 4
 mode:
   .zero 4
+
+/* Output buffer for the resulting, recovered message. */
+.globl out_buf
+.balign 512
```

Comment:
I do kinda wonder if there should be a constant like `OTBN_DATA_ALIGN` or something in some header... I don't think we have a good place to put that though. =(

Reply:
Yeah, it would be nice to get a warning in an easier way than extracting the OTBN error bits when the test fails, or even better have the data automatically aligned when needed. But I also don't know how that would work exactly.